app/build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id 'application'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation libs.junit.jupiter
+
+    testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
+
+    implementation libs.guava
+}
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(22)
+    }
+}
+
+application {
+    mainClass = 'dev.andrylat.app.App'
```

Comment:
Packaging is wrong.

company is andrylat.dev. Here everything is fine.
But in my company I have multiple projects (students).
Each project has subproject (students have different projects).
Fix the structure to follow the requirement.